packages.json

```diff
@@ -11148,5 +11148,18 @@
     "description": "A Beautiful Markdown Parser in the Nim World.",
     "license": "MIT",
     "web": "https://github.com/soasme/nim-markdown"
+  },
+  {
+    "name": "sugerror",
+    "url": "https://github.com/quelklef/nim-sugerror",
+    "method": "git",
+    "tags": [
+      "error",
+      "errors",
+      "error handling"
```

Comment:
no spaces in tags

Reply:
doesn't nimble have a builtin linter for tags?